lib/stdlib/putchar.c

```diff
@@ -15,6 +16,8 @@
 int putchar(int c)
 {
 	int res;
+	if (COREBOOT)
```

Comment:
[whoops, I thought I commented this last time already]

I always prefer a C if() over a preprocessor #if even for macro constants where possible, because I feel it makes the code flow easier to read and it allows the compiler to syntax check both branches (so it's harder for bit rot to creep into code paths of rarely used configurations when stuff gets refactored over the years). But if official Trusted Firmware style says to always use #if for those cases I can change this, of course.